src/Data/Nat/DivMod.agda

```diff
@@ -243,39 +226,50 @@ m/n<m m n n≥2 = *-cancelʳ-< _ (m / n) m (begin-strict
               o ∣ m → o ∣ n → m / o ≡ n / o → m ≡ n
 /-cancelʳ-≡ {m} {n} {o} o∣m o∣n m/o≡n/o = begin-equality
   m           ≡⟨ m*[n/m]≡n {o} {m} o∣m ⟨
-  o * (m / o) ≡⟨  cong (o *_) m/o≡n/o ⟩
-  o * (n / o) ≡⟨  m*[n/m]≡n {o} {n} o∣n ⟩
+  o * (m / o) ≡⟨ cong (o *_) m/o≡n/o ⟩
+  o * (n / o) ≡⟨ m*[n/m]≡n {o} {n} o∣n ⟩
   n           ∎
 
 m<n⇒m/n≡0 : ∀ {m n} .{{_ : NonZero n}} → m < n → m / n ≡ 0
 m<n⇒m/n≡0 {m} {suc n-1} (s≤s m≤n) = divₕ-finish n-1 m n-1 m≤n
 
 m≥n⇒m/n>0 : ∀ {m n} .{{_ : NonZero n}} → m ≥ n → m / n > 0
 m≥n⇒m/n>0 {m@(suc _)} {n@(suc _)} m≥n = begin
-  1     ≡⟨ sym (n/n≡1 m) ⟩
+  1     ≡⟨ n/n≡1 m ⟨
   m / m ≤⟨ /-monoʳ-≤ m m≥n ⟩
   m / n ∎
 
+m/n≡0⇒m<n : ∀ {m n} .{{_ : NonZero n}} → m / n ≡ 0 → m < n
+m/n≡0⇒m<n {m} {n@(suc _)} m/n≡0  with <-≤-connex m n
+... | inj₁ m<n = m<n
+... | inj₂ n≤m = contradiction m/n≡0 (≢-nonZero⁻¹ _)
+  where instance _ =  >-nonZero (m≥n⇒m/n>0 n≤m)
+
+m/n≢0⇒n≤m : ∀ {m n} .{{_ : NonZero n}} → .{{NonZero (m / n)}} → n ≤ m
```

Comment:
Eek I'm not sure about this one. I can't help but feel it should be `m / n ≢ 0` rather than `.{{NonZero (m / n)}}` and I'm trying to work out why. Possible reasons:
- Symmetry with the lemma above
- We're never generating proofs of inequalities via instance search so no need to make it an instance.

If you're keen to still have it in this form then we probably need to make `m` and `n` explicit...

Reply:
Yes, this is an odd one. Better to revert it for now, I guess.

Reply:
Just a heads up that the incoming commit hasn't seemed to arrived?

Reply:
Oh?

Reply:
Ah, yup. Here now. Thanks!